fe-local-treasures/screens/ForgetPassword.jsx

```diff
@@ -0,0 +1,70 @@
+import React, {useState} from 'react'
```

Comment:
file called forgEt password but component called ForgOt password, would be good to change